### PR TITLE
Add tests for expiry

### LINKED ISSUE #94: Add more test for addCrypto and addForex parser
- Currently these 2 dont have tests for expiry for in the past expiry and invalid date inputs. To be started once #91 closes

### PR DESCRIPTION
**Pull request by KVignesh122**
Add more tests on expiry for addForexParser and addCryptoParser

Fix #94

### CODE REVIEW

File: src/test/java/seedu/mtracker/console/AddForexParserTest.java [hunk 1]
```diff
@@ -74,6 +70,29 @@
...
 
+    public static final String USER_INPUT_TRY_INVALID_EXPIRY = SEPARATOR_SPECIFIER + "TTTXXX"
+            + SEPARATOR_SPECIFIER + "0.81"
+            + SEPARATOR_SPECIFIER + "foobar"
+            + SEPARATOR_SPECIFIER + "negative"
+            + SEPARATOR_SPECIFIER + "0.79"
+            + SEPARATOR_SPECIFIER + "0.70"
+            + SEPARATOR_SPECIFIER.repeat(2) + "testDate"
+            + SEPARATOR_SPECIFIER + "31/12/2021"
+            + SEPARATOR_SPECIFIER + "2021-01-01"
```
Comment (theodorekwok) [inline src/test/java/seedu/mtracker/console/AddForexParserTest.java:81]: Use past date here

**Review by theodorekwok — COMMENTED** [review]
Looks good, can delete future date and days difference in addcryptoparsertest

File: src/test/java/seedu/mtracker/console/AddForexParserTest.java (hunk 1 above)
Comment (KVignesh122) [reply src/test/java/seedu/mtracker/console/AddForexParserTest.java:81]: Sorry I wanted to put "2021.01.01" actually. PAST_DATE is used in USER_INPUT_TRY_PAST_EXPIRY below.

**Comment by theodorekwok** [comment]
LGTM!

**Pull request merged**